src/taipy/core/_scheduler/_dispatcher/_development_job_dispatcher.py

```diff
@@ -9,14 +9,24 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-from ._standalone_job_dispatcher import _StandaloneJobDispatcher
+from .._abstract_scheduler import _AbstractScheduler
 from .._executor._synchronous import _Synchronous
+from ._standalone_job_dispatcher import _StandaloneJobDispatcher
 
 
 class _DevelopmentJobDispatcher(_StandaloneJobDispatcher):
     """Manages job dispatching (instances of `Job^` class) in a synchronous way."""
 
-    def __init__(self):
-        super().__init__()
-        self._executor = _Synchronous()
+    def __init__(self, scheduler: _AbstractScheduler):
+        super().__init__(scheduler)
```

Comment:
?

Reply:
I'm passing the scheduler to the parent level where it will be stored in self.scheduler